platform/posix/iot_logging.c

```diff
@@ -28,10 +28,15 @@
 /* Standard includes. */
 #include <stdarg.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
+#include <assert.h>
 
 /* Platform clock include. */
-#include "platform/iot_clock.h"
+#include "platform/include/iot_clock.h"
```

Comment:
Does iot_clock.h need the "config.h" header?

Reply:
Removed.